### ISSUE TITLE
remove -Bsymbolic-functions from debbuild

### ISSUE THREAD

**Issue by gerkey** [ros/catkin#72]
IIRC, it's this little gem in our debian rules file that we specify
with our source debs:

ros_release/rosdeb/resources/source_deb/rules:

```
 LDFLAGS="" $(MAKE) ROS_DEBBUILD=1 ROS_DESTDIR=$(CURDIR)/debian/${ROS_STACK_DEBIAN_NAME} SUBSTFILE=$(CURDIR)/debian/${ROS_STACK_DEBIAN_NAME}.substvars install
```

Something in catkin-generate-debian, I assume

**Comment by gerkey** [comment]
pbuilder environment setup on blw; testing underway there.

**Comment by gerkey** [comment]
Added LDFLAGS override, 88dcc4e.  Tested on ros-fuerte-ros 1.7.11-1 release, and it seems to work.

**Comment by gerkey** [comment]
Released the change in catkin 0.3.15
